x-pack/plugins/apm/server/routes/source_maps/route.ts

```diff
@@ -40,14 +40,28 @@ export type SourceMap = t.TypeOf<typeof sourceMapRt>;
 const listSourceMapRoute = createApmServerRoute({
   endpoint: 'GET /api/apm/sourcemaps',
   options: { tags: ['access:apm'] },
+  params: t.partial({
+    query: t.partial({
+      page: toNumberRt,
+      perPage: toNumberRt,
+    }),
+  }),
```

Comment:
Like the comment above, why don't make these queries required?

Reply:
It's a public API so it would be a breaking change